### PR TITLE
Fixes `cpu`

### PR DESCRIPTION
**Pull request by karsa-mistmere**
This PR fixes design guideline violations of the `cpu` icon.
image
(Note: the central rectangle remains unrounded)

### CODE REVIEW

**Comment by ericfennis** [comment]
What do you think of this?
image

**Comment by karsa-mistmere** [comment]
Yeah, that also LGTM.

**Comment by ericfennis** [comment]
@karsa-mistmere I think we need some more opinions on this. Curious what other people think. I'm fine with both 😆

**Comment by karsa-mistmere** [comment]
This is also maybe an option:
image

100%:
image

200%:
image



Right now, I think `cpu-2` (yours) looks best compared to square for example.

**Comment by ericfennis** [comment]
yeah you right, its nice to have it consistant.

File: icons/cpu.svg [hunk 1]
```diff
@@ -9,14 +9,14 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="4" y="4" width="16" height="16" rx="2" ry="2" />
+  <rect x="5" y="5" width="14" height="14" rx="2" ry="2" />
```
Comment (ericfennis) [inline icons/cpu.svg:12]:
```suggestion
  <rect x="4" y="4" width="16" height="16" rx="2" ry="2" />
```

File: icons/cpu.svg [hunk 2]
```diff
@@ -9,14 +9,14 @@
...
-  <line x1="1" y1="9" x2="4" y2="9" />
-  <line x1="1" y1="14" x2="4" y2="14" />
+  <line x1="9" y1="2" x2="9" y2="5" />
+  <line x1="15" y1="2" x2="15" y2="5" />
+  <line x1="9" y1="19" x2="9" y2="22" />
+  <line x1="15" y1="19" x2="15" y2="22" />
+  <line x1="19" y1="9" x2="22" y2="9" />
+  <line x1="19" y1="14" x2="22" y2="14" />
+  <line x1="2" y1="9" x2="5" y2="9" />
+  <line x1="2" y1="14" x2="5" y2="14" />
```
Comment (ericfennis) [inline icons/cpu.svg:21]:
```suggestion
  <line x1="9" y1="2" x2="9" y2="4" />
  <line x1="15" y1="2" x2="15" y2="4" />
  <line x1="9" y1="21" x2="9" y2="22" />
  <line x1="15" y1="20" x2="15" y2="22" />
  <line x1="20" y1="9" x2="22" y2="9" />
  <line x1="20" y1="14" x2="22" y2="14" />
  <line x1="2" y1="9" x2="4" y2="9" />
  <line x1="2" y1="14" x2="4" y2="14" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Here you go.

**Review by ericfennis — APPROVED** [review]

**Pull request merged**